pkg/beacon/beacon.go

```diff
@@ -111,7 +128,24 @@ func Initialize(
 			)
 		}
 
+		newEntry := event.NewEntry.Text(16)
 		go func() {
+			if ok := groupSelected.Add(newEntry); !ok {
+				logger.Errorf(
+					"group selection with seed [0x%v] has already started",
```

Comment:
`group selection event with seed [0x%x] already registered`? 

We should also take the opportunity and replace `%v` with `%x` in the info log below.

Reply:
done